conda/core/prefix_data.py

```diff
@@ -385,7 +386,7 @@ def remove(self, package_name):
 
         del self._prefix_records[package_name]
 
-    def get(self, package_name, default=NULL):
+    def get(self, package_name: str, default: Any = NULL) -> PackageRecord | Any:
```

Comment:
Is there value in being more explicit?

```suggestion
    T = TypeVar("T")  # this should be defined above in the `if TYPE_CHECKING` clause
    def get(self, package_name: str, default: T = NULL) -> PackageRecord | T:
```